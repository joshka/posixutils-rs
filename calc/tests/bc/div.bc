scale = 1
a = 1/2
scale(a)
a

scale = 0
a = 1/2
scale(a)
a

scale = 10
a = 1/2
scale(a)
a

scale = 10
4 / 2
10 / 5
3.4578 / 20
1234 / 245.8909

quit
