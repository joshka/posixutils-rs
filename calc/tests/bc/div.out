1
0.5
0
0
10
0.5000000000
2.0000000000
2.0000000000
0.1728900000
5.0184858406
